### PR TITLE
no squashing the weekly merge

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Don't squash the weekly merge to master (and therefore, no squashing master overwrite of weekly).

### Resolves:

No JIRA.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ Create a production build of the `weekly` branch, merge it, then reset `weekly`.
     [okta.github.io weekly-2017.25] git push origin weekly-2017.25:weekly-2017.25
     ```
 
-6. After it's been approved and passes the Travis tests, merge it into `master`. The changes are live! Don't merge into `weekly` before the next step!
+6. After it's been approved and passes the Travis tests, merge it into `master` **without squashing**. The changes are live! Don't merge into `weekly` before the next step!
```
Comment (richardmateosian-okta) [inline README.md]: The change is fine, but the unchanged line "Don't merge into `weekly` before the next step!" could be more explicit about the fact that nobody should merge into weekly. And implicit in idea is the question of how everyone knows they shouldn't do that.

Comment (mystiberry-okta) [inline README.md]: That's not true AFAIK. Will follow up.

**Comment by mystiberry-okta** [comment]
Hold on this--it's a little more complicated than I thought.

**Comment by mystiberry-okta** [comment]
This is likely not the right solution. Waiting for input to doc any needed change.

**Pull request closed without merging**